python/dgl/dataloading/dataloader.py

```diff
@@ -791,6 +845,11 @@ class DataLoader(torch.utils.data.DataLoader):
         Whether to pin the feature tensors into pinned memory.
 
         Default: True if the graph is on CPU and :attr:`device` is CUDA.  False otherwise.
+    gpu_cache : dict or tuple(dict, dict), optional
```

Comment:
Add link `https://github.com/NVIDIA-Merlin/HugeCTR/blob/main/gpu_cache/ReadMe.md` for further reference.